options/options.js

```diff
@@ -19,6 +19,22 @@ function restoreOptions() {
   var value = '';
   var getting = '';
 
+  // set event click for "move-up" and "move-down" and set the text of those buttons
+  document.querySelectorAll('button[id]').forEach(function(item) {
+    if (item.id.endsWith("move-up")) {
+      document.getElementById(item.id).addEventListener('click', function () {
+        moveUpDown(item.id, "up");
+      });
+      document.getElementById(item.id).innerHTML="▲";
+    }
+    else if(item.id.endsWith("move-down")) {
+      document.getElementById(item.id).addEventListener('click', function () {
+        moveUpDown(item.id, "down");
+      });
+      document.getElementById(item.id).innerHTML="▼";
```

Comment:
```
document.querySelectorAll('button[id]').forEach(function(item) {
    document.getElementById(item.id).addEventListener('click', function () {
      if (item.id.endsWith("move-up")) {
        moveUpDown(item.id, "up");
        return;
      } 
      moveUpDown(item.id, "down");
    });
  });
```
With that change this now works

Reply:
So you can change the symbol fast and easilier (maybe also remove it and replace it with image).

Reply:
Yes but the button doesn't move and there are two, so is better to define them as default and simplify the code.
Probably is more easy to use css with `content` attribute to edit the html.

Reply:
As you prefer 😄 I wasn't sure about that symbol, so I would like your opinion. 👍